frontend/public/i18n.js

```diff
@@ -112,11 +112,11 @@ i18n
       },
       saveMissing: true,
       missingKeyHandler: function(lng, ns, key) {
-        window.windowError = new Error(
-          `Missing i18n key "${key}" in namespace "${ns}" and language "${lng}."`,
-        );
+        const error = new Error();
+        error.message = `Missing i18n key "${key}" in namespace "${ns}" and language "${lng}."`;
+        window.windowError = error;
```

Comment:
I'm tempted to make this PR a WIP because I don't understand why the original way doesn't work.  Using the original `new Error(...)` constructor resulted in a defined, but empty `window.windowError` object when processed in updated `protractor.conf.ts`'s `formatWindowError`.
What's even weirder is I could add `error.name, error.fooBar` and those props would be there in `formatWindowError`, but `error.stack` would disappear!
I suspect it has something to do with `Error` typescripting..maybe..?  In my IDE when I click on `Error` it comes up with two different defs, one in `node`:
```
interface Error {
    stack?: string;
}
```
..and one in `typescript.es5`:
```
interface Error {
    name: string;
    message: string;
    stack?: string;
}
```
`stack` is in both defs, so really confused as to why it doesn't 'stick'!?
```